meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -221,6 +221,10 @@ FEATURE_PACKAGES_multilib-runtime ?= "${@d.getVar('MULTILIB_RUNTIME_FEATURE_PACK
 DISTRO_EXTRA_RRECOMMENDS += "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', '', 'nss-myhostname', d)}"
 ## }}}1
 ## Workarounds & Overrides {{{1
+# Re-export and exclude from checksums
```

Comment:
just out of curiosity, why do we need to export and exclude it but not other variables like SALT ones?

Reply:
We do, actually, just those ones are handled in meta-sourcery, since that's where our support for our external toolchains lie, and that's where we need to support salt. WSL_INTEROP is actually not toolchain specific, since it'd be needed to run any windows host executable.

Reply:
See https://github.com/MentorEmbedded/meta-sourcery/pull/245